paddle/framework/threadpool.cc

```diff
@@ -14,6 +14,8 @@
 
 #include "paddle/framework/threadpool.h"
 
```

Comment:
Please remove the redundant blank line between two `#include`.

Reply:
This is not a useless blank like. Please refer to examples in https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Got it, thanks!